cw/cw305/util/device.py

```diff
@@ -0,0 +1,84 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+r"""CW305 utility functions. Used to configure FPGA with OpenTitan design."""
+
+import subprocess
+import time
+
+import chipwhisperer as cw
+
+
+SPIFLASH=r'bin/linux/spiflash'
+
+
+class OpenTitan(object):
+  def __init__(self, bitstream, fw_image, pll_frequency, baudrate):
+      self.bitstream = bitstream
+      self.fw_image = fw_image
+      self.pll_frequency = pll_frequency
+      self.initialized = False
+      self.fpga = None
+      self.scope = None
+      self.target = None
+      self.baudrate = baudrate
+
+  def initialize(self):
```

Comment:
Is there a reason why this can not be inlined in `__init__`?

Reply:
Updated to follow the approach you suggested below.